lib/pbench/server/database/models/active_tokens.py

```diff
@@ -1,51 +1,55 @@
-import datetime
+from datetime import datetime
+from typing import TypeVar
 
 from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
 
 from pbench.server.database.database import Database
 
+AT = TypeVar("AT", bound="ActiveToken")
 
-class ActiveTokens(Database.Base):
-    """Token model for storing the active auth tokens at any given time"""
+
+class ActiveToken(Database.Base):
```

Comment:
While I'm hesitant to say this because I appreciate the lack of extra churn, when renaming the SQL object it would have been more consistent to rename the file as well. 👀 

(Then again, this and `User` go away when we finish the Keycloak transition...)

Reply:
I agree with Dave, since you are at it you might change the filename as well.